docs/serving/samples/hello-world/helloworld-nodejs/Dockerfile

```diff
@@ -14,7 +14,7 @@ COPY package*.json ./
 RUN npm install --only=production
 
 # Copy local code to the container image.
-COPY . .
+COPY . ./
```

Comment:
This is a syntax preference.

2 pros
- All destination folders end with a slash (doc consistency).
- This will prevent some developers from falling into this dockerfile syntax trap where `COPY file1 .` works but `COPY file1 file2 .` fails. Additionally, the sample pattern `COPY *.ext .` (without `/`) will eventually bite (only works if there's exactly 1 `.ext` file). This is used with `*.csproj` in the C# sample. Using `COPY *.ext ./` also prevents this kind of issues from appearing indirectly.

1 con
- `COPY . .` looks nicer than `COPY . ./`.

Reply:
Added a [comment](https://github.com/knative/docs/pull/1732/commits/4a14b3da170cd8d8a49da166986a93a83e952905#r319433025) in #1732 for the potential issue with `COPY *.csproj .`.